README.md

```diff
@@ -57,23 +53,35 @@ If you use any of these files, please carefully follow the [instructions for cit
 To prepare the environment to run the PhenoPLIER code, follow the steps in [environment](environment/).
 This will create a conda environment and download the necessary data.
 Depending on your Internet speed, this shouldn't take more than 24 hours.
+We tested the code in Ubuntu 20.04+.
```

Comment:
Since it sounds like you did run the code in 18.04, do you want to say you tested it in both? Give yourself a little credit for it? :joy:

Reply:
Well, I started with Ubuntu 18.04, but then I quickly switched to 20.04, and then to 22.04. Most of the analyses were not run using version 18.04, so I feel it's not true to say that anymore :) It should work on all of them though.